### PR TITLE
Additional tests for create command

### PR DESCRIPTION
**Pull request by kuter**

### CODE REVIEW

File: tests/management/commands/test_create_command.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    """Tests for create_command command."""
+
+    def setUp(self):  # noqa
+        self.management_command_path = os.path.join(
+            settings.BASE_DIR, 'tests/testapp/management')
+        self.command_template_path = os.path.join(
+            settings.BASE_DIR, 'django_extensions/conf/command_template')
+
+        self.files = ['__init__.py', 'commands/__init__.py',
+                      'commands/sample.py']
```
Comment (trbs) [inline tests/management/commands/test_create_command.py]: could you please either use a single line for this or use the following style:

```
self.files = [
    '__init__.py', 
    'commands/__init__.py',
    'commands/sample.py',
]
```

Same goes for the other lines in this file.
We allow lines to be pretty wide and don't hold to 80 chars wide :-)

**Comment by trbs** [comment]
Are all the `# noqa` 's needed ?

All feedback are style related btw, the PR content wise is good to merge 👍

File: tests/management/commands/test_create_command.py (hunk 1 above)
Comment (kuter) [reply tests/management/commands/test_create_command.py]: I like that style too, changed.

**Comment by kuter** [comment]
> Are all the `# noqa` 's needed ?

Generaly I using noqa to get rid of pep257 and flake8 warnings (mainly line length and missing docstring D102).

**Comment by trbs** [comment]
> Generaly I using noqa to get rid of pep257 and flake8 warnings (mainly line length and missing docstring D102).

Flake8 should not complain about line length... in `setup.cfg` it is set to `max-line-length = 512`.
I've quickly looked at `pydocstyle` (pip257) and does not seem to pickup the settings in `setup.cfg` for me, probably since I already have a `~/.pydocstylerc`.

I committed some more lines to help with the `D1??` warnings and partial fixes for other docstyle warnings in the codebase.

**Pull request merged**